src/api/libcellml/generator.h

```diff
@@ -78,8 +78,7 @@ class LIBCELLML_EXPORT GeneratorVariable
      * @c Component in which the @c Variable is first defined (in the case of
      * the variable of integration), initialised (in the case of a constant) or
      * actually computed (in the case of a state, computed constant or algebraic
-     * variable). It may or may not be the same @c Component as the parent
-     * component of the @c Variable.
+     * variable).
```

Comment:
3) Because of points 1 and 2, the `component()` is always going to be the parent component of `variable()`, which makes our API better than before.